Dockerfile

```diff
@@ -26,11 +26,11 @@ RUN useradd -M cozy \
 
 
 # Need ENV VARS:
-ENV NODE_ENV production \
```

Comment:
sorry, I forgot the '=', it should be `NODE_ENV=production`.
Can you update your PR please? 
It is just that with 5 lines, it is creating 5 layers...